frontend/libs/studio-components/src/components/StudioContentMenu/context/StudioContentMenuContext.test.tsx

```
@@ -0,0 +1,47 @@
+    render(
+      <StudioContentMenuContextProvider>
+        <button>My button</button>
+      </StudioContentMenuContextProvider>,
+    );
+
+    expect(screen.getByRole('button', { name: 'My button' })).toBeInTheDocument();
+  });
+
+  it('should provide a useStudioContentMenuContext hook', () => {
+    const TestComponent = (): ReactElement => {
+      const {} = useStudioContentMenuContext();
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)